### PR TITLE
Modernization

### PR DESCRIPTION
**Pull request by kaimallea**
Some minor modernization tweaks. Most important change here is the mocking that makes uploadFile testable

cc @KenEucker if you still have an interest (no pressure)

### CODE REVIEW

File: jest.setup.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+const mockfs = require('mock-fs');
+
+// Establish API mocking before all tests.
+beforeAll(() => server.listen());
+
+beforeEach(() => {
+  mockfs({
+    '/home/user/meme.jpg': Buffer.from([8, 6, 7, 5, 3, 0, 9]),
+    '/home/user/lol.jpg': Buffer.from([9, 0, 3, 5, 7, 6, 8]),
+  });
```
Comment (kaimallea) [inline jest.setup.js:11]: mock file system available in all tests. any fs.* method will think these are real. key == file, value == file contents

File: lib/mocks/handlers.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        id: 'JK9ybyj',
+        deletehash: 'j83zimv4VtDA0Xp',
+        link: 'https://i.imgur.com/JK9ybyj.jpg',
+      },
+      success: true,
+      status: 200,
+    };
+    return res(ctx.json(response));
+  }),
+];
```
Comment (kaimallea) [inline lib/mocks/handlers.js:16]: mock response handler used for `uploadFile` test. As easy as adding more handlers to mock responses for other methods.

**Review by KenEucker — APPROVED** [review]
@kaimallea looks good to me!

I was able to successfully run all of the tests on my machine.

**Pull request merged**